examples/usvg_viewer/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "usvg-viewer"
```

Comment:
So far, I've kept us consistently using snake_case example names. I'd rather keep doing that, especially since the directory name is already using the snake_case convention.